csharp/src/Google.Protobuf.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.22823.1
+# Visual Studio 2013
+VisualStudioVersion = 12.0.31101.0

```

Comment:
Just as a word of warning, I'd still like to update the runtime to use C# 6 code at some point soon, which will require VS2015. (I think the version of Mono we're using supports *most* of C# 6.)